client/rest-high-level/src/main/java/org/elasticsearch/client/transform/transforms/TransformStats.java

```diff
@@ -117,7 +117,7 @@ public boolean equals(Object other) {
 
     public enum State {
 
-        STARTED, INDEXING, ABORTING, STOPPING, STOPPED, FAILED;
+        STARTED, INDEXING, ABORTING, STOPPING, STOPPED, FAILED, WAITING;
```

Comment:
Is it going to be obvious to the user what is the transform waiting for?

Reply:
from _only_ looking at the state, probably not. But you get the header warning and the reason field that contains more information.